### PR TITLE
Add new afternoonlabs/breeze/rev0 split keyboard

### PR DESCRIPTION
**Pull request by eithanshavit**
Adding a new keyboard named Breeze under the AfternoonLabs brand.

## Description

Breeze Rev0 is a split keyboard with 6×4 keys, 4 key thumb cluster, arrow keys, and a 6 key macro cluster, with ortholinear column-staggered.

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/afternoonlabs/breeze/rev0/config.h [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+
+#pragma once
+
+/* USB Device descriptor parameter */
+#define VENDOR_ID 0x616C
+#define PRODUCT_ID 0x0001
+#define DEVICE_VER 0x0000
+#define MANUFACTURER AfternoonLabs
+#define PRODUCT Breeze
+#define DESCRIPTION Breeze Rev0 is a split keyboard with 6×4 keys, 4 key thumb cluster, arrow keys, and a 6 key macro cluster, with ortholinear column - staggered.
```
Comment (drashna) [inline keyboards/afternoonlabs/breeze/rev0/config.h:25]: This setting is not used anymore and should be removed.
```suggestion
```

File: keyboards/afternoonlabs/breeze/rev0/rules.mk [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+CONSOLE_ENABLE = yes       # Console for debug
+COMMAND_ENABLE = no        # Commands for debug and configuration
+NKRO_ENABLE = no           # Nkey Rollover - if this doesn't work, see here: https://github.com/tmk/tmk_keyboard/wiki/FAQ#nkro-doesnt-work
+BACKLIGHT_ENABLE = no      # Enable keyboard backlight functionality
+MIDI_ENABLE = no           # MIDI controls
+AUDIO_ENABLE = no          # Audio output on port C6
+UNICODE_ENABLE = no        # Unicode
+BLUETOOTH_ENABLE = no      # Enable Bluetooth with the Adafruit EZ-Key HID
+RGBLIGHT_ENABLE = no       # Enable WS2812 RGB underlight.
+SLEEP_LED_ENABLE = no      # Breathing sleep LED during USB suspend
```
Comment (drashna) [inline keyboards/afternoonlabs/breeze/rev0/rules.mk:30]:
```suggestion
AUDIO_ENABLE = no          # Audio output
UNICODE_ENABLE = no        # Unicode
BLUETOOTH_ENABLE = no      # Enable Bluetooth
RGBLIGHT_ENABLE = no       # Enable WS2812 RGB underlight.
SLEEP_LED_ENABLE = no      # Breathing sleep LED during USB suspend
```

**Review by drashna — CHANGES_REQUESTED** [review]

File: keyboards/afternoonlabs/breeze/rev0/rules.mk (hunk 2 above)
Comment (eithanshavit) [reply keyboards/afternoonlabs/breeze/rev0/rules.mk:30]: Thanks @drashna!

File: keyboards/afternoonlabs/breeze/config.h [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+#pragma once
+
+#include "config_common.h"
+
+#ifndef NO_ACTION_MACRO
+#    define NO_ACTION_MACRO
+#endif
+#ifndef NO_ACTION_FUNCTION
+#    define NO_ACTION_FUNCTION
+#endif
```
Comment (drashna) [inline keyboards/afternoonlabs/breeze/config.h:26]:
```suggestion
#define NO_ACTION_MACRO
#define NO_ACTION_FUNCTION
```

**Review by drashna — CHANGES_REQUESTED** [review]
The empty rules.mk file should be removed from the default keymap folder.

File: keyboards/afternoonlabs/breeze/config.h (hunk 3 above)
Comment (eithanshavit) [reply keyboards/afternoonlabs/breeze/config.h]: Thanks again @drashna, I've resolved your comments.

**Comment by eithanshavit** [comment]
Thanks for the feedback, all recent change request addressed. @drashna, anything else I should consider?

**Review by drashna — APPROVED** [review]

File: keyboards/afternoonlabs/breeze/keymaps/default/readme.md [hunk 4]
```diff
@@ -0,0 +1,5 @@
+![Breeze Layout Image](https://imgur.com/a/1BKNmyR)
+
+# Default Breeze Layout
```
Comment (fauxpark) [inline keyboards/afternoonlabs/breeze/keymaps/default/readme.md:3]:
```suggestion
# Default Breeze Layout

![Breeze Layout Image](https://i.imgur.com/O8FJWD6.jpeg)
```

File: keyboards/afternoonlabs/breeze/rev0/readme.md [hunk 5]
```diff
@@ -0,0 +1,15 @@
+# Breeze
+
+![Breeze](https://imgur.com/a/1BKNmyR)
```
Comment (fauxpark) [inline keyboards/afternoonlabs/breeze/rev0/readme.md:3]:
```suggestion
# Breeze

![Breeze](https://i.imgur.com/O8FJWD6.jpeg)
```

File: keyboards/afternoonlabs/breeze/rev0/rules.mk [hunk 6]
```diff
@@ -0,0 +1,31 @@
...
+MCU = atmega32u4
+
+# Bootloader selection
+#   Teensy       halfkay
+#   Pro Micro    caterina
+#   Atmel DFU    atmel-dfu
+#   LUFA DFU     lufa-dfu
+#   QMK DFU      qmk-dfu
+#   ATmega32A    bootloadHID
+#   ATmega328P   USBasp
```
Comment (fauxpark) [inline keyboards/afternoonlabs/breeze/rev0/rules.mk:11]:
```suggestion
```

File: keyboards/afternoonlabs/breeze/rev0/rules.mk [hunk 7]
```diff
@@ -0,0 +1,31 @@
...
+COMMAND_ENABLE = no        # Commands for debug and configuration
+NKRO_ENABLE = no           # Nkey Rollover - if this doesn't work, see here: https://github.com/tmk/tmk_keyboard/wiki/FAQ#nkro-doesnt-work
+BACKLIGHT_ENABLE = no      # Enable keyboard backlight functionality
+MIDI_ENABLE = no           # MIDI controls
+AUDIO_ENABLE = no          # Audio output
+UNICODE_ENABLE = no        # Unicode
+BLUETOOTH_ENABLE = no      # Enable Bluetooth
+RGBLIGHT_ENABLE = no       # Enable WS2812 RGB underlight.
+SLEEP_LED_ENABLE = no      # Breathing sleep LED during USB suspend
+SPLIT_KEYBOARD = yes
```
Comment (fauxpark) [inline keyboards/afternoonlabs/breeze/rev0/rules.mk:31]:
```suggestion
#   change yes to no to disable
#
BOOTMAGIC_ENABLE = lite     # Virtual DIP switch configuration
MOUSEKEY_ENABLE = no        # Mouse keys
EXTRAKEY_ENABLE = no        # Audio control and System control
CONSOLE_ENABLE = yes        # Console for debug
COMMAND_ENABLE = no         # Commands for debug and configuration
# Do not enable SLEEP_LED_ENABLE. it uses the same timer as BACKLIGHT_ENABLE
SLEEP_LED_ENABLE = no       # Breathing sleep LED during USB suspend
# if this doesn't work, see here: https://github.com/tmk/tmk_keyboard/wiki/FAQ#nkro-doesnt-work
NKRO_ENABLE = no            # USB Nkey Rollover
BACKLIGHT_ENABLE = no       # Enable keyboard backlight functionality
RGBLIGHT_ENABLE = no        # Enable keyboard RGB underglow
BLUETOOTH_ENABLE = no       # Enable Bluetooth
AUDIO_ENABLE = no           # Audio output
SPLIT_KEYBOARD = yes
```

**Review by fauxpark — CHANGES_REQUESTED** [review]

**Comment by eithanshavit** [comment]
Thanks for the suggestions @fauxpark, I've updated the PR with the changes.

**Review by fauxpark — APPROVED** [review]

**Comment by drashna** [comment]
Thanks!

**Pull request merged**